server/openapi/model_linter_result.go

```diff
@@ -10,6 +10,8 @@
 package openapi
 
 type LinterResult struct {
+	MinimumScore int32 `json:"minimumScore,omitempty"`
```

Comment:
We are now going to persis the minimum score that was used during the execution, not using the global